docs/core/tools/global-tools-how-to-create.md

```diff
@@ -0,0 +1,197 @@
+---
```

Comment:
I think the file name could be create-global-tools.md

Reply:
@mairaw Do we have a naming convention? The one I used in Azure was something like [feature]-[article-type]-[verb]

This helped group-like articles in the directory list.

Reply:
@mairaw ping :)